GPU/Common/FramebufferCommon.cpp

```diff
@@ -700,7 +700,7 @@ void FramebufferManagerCommon::UpdateFromMemory(u32 addr, int size, bool safe) {
 						// If we're not rendering to it, format may be wrong.  Use displayFormat_ instead.
 						fmt = displayFormat_;
 					}
-					DrawPixels(vfb, 0, 0, Memory::GetPointer(addr | 0x04000000), fmt, vfb->fb_stride, vfb->width, vfb->height);
+					DrawPixels(vfb, 0, 0, Memory::GetPointer(addr), fmt, vfb->fb_stride, vfb->width, vfb->height);
```

Comment:
This is inside a `MaskedEqual()` check which is probably dangerous and needs to be changed.  I think `MaskedEqual` needs removing entirely, or it needs to check the VRAM bit and not mask it out (probably better to just ensure we never put anything in with kernel/cached bits.)

-[Unknown]

Reply:
Removing MaskedEqual entirely. I think this should be fine as-is now.